### DISCUSSION TITLE
New issues when?

### DISCUSSION THREAD

**Discussion by lrnzdc3696** [HuyaneMatsu/hata#61] [Q&A] [answered]
We need more issues. I want to contribute but there are no issues to be fixed...

(ノT＿T)ノ ＾┻━┻

**Comment by HuyaneMatsu** [comment, marked as answer]
Make issues then? 👀

**Comment by HuyaneMatsu** [comment]
There is a new [issue](https://github.com/HuyaneMatsu/hata/issues/65). Hope you can make a pr to resolve it 🙏🏻
